src/inject/dynamic-theme/uuid.ts

```diff
@@ -0,0 +1,9 @@
+
+export function generateUID() {
+    const s = window.crypto.getRandomValues(new Uint8Array(16));
+    return s[0].toString(16) + s[1].toString(16) + s[2].toString(16) + s[3].toString(16) +
```

Comment:
Everything was OK with your `hexify` function, except that you could make it shorter like `(x < 16 ? '0' : '') + x.toString(16)`. Now you can have a situation when `f + ff == ff + f`.

If you decide to make it not compatible with the spec, you can simply use `Array.from(...).map(hexify).join('') to not repeat the same expression 16 times.